src/Command/Configuration.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Roave\BackwardCompatibility\Command;
+
+use Psl\Json;
+use Psl\Type;
+use Roave\BackwardCompatibility\Baseline;
+use RuntimeException;
+
+final class Configuration
```

Comment:
Do we need to make this `@psalm-immutable`, or does Psalm infer that these days? I genuinely don't know these days :grin:

Reply:
I'm on the same boat but it doesn't hurt to add it :rofl: